src/main/java/cms/gov/madie/measure/services/QdmPackageService.java

```diff
@@ -33,7 +33,7 @@ public class QdmPackageService implements PackageService {
 
   @Override
   public PackageDto getMeasurePackage(
-      Measure measure, String accessToken, boolean includeElmWarnings) {
+      Measure measure, boolean includeElmWarnings, String accessToken) {
     if (!measure.getMeasureMetaData().isDraft()) {
       Optional<Export> savedExport = repository.findByMeasureId(measure.getId());
```

Comment:
We have to return a different object here based the includeElmWarnings flag for versioned measures

Reply:
in adddition to .exportPackage(savedExport.get().getPackageData())
we have to return .exportPackage(savedExport.get().getPublishablePackageData()) when includeElmWarnings is false